modules/virt-creating-configmap.adoc

```diff
@@ -289,18 +289,14 @@ $ oc edit clusterserviceversion -n openshift-cnv kubevirt-hyperconverged-operato
 +
 [source,yaml]
 ----
-            ...
-            spec:
-              containers:
-              - env:
-                ...
-                - name: OS_CONFIGMAP_NAME
-                  value: os-configmap <1>
-                - name: OS_CONFIGMAP_NAMESPACE
-                  value: default <2>
+apiVersion: v1
```

Comment:
The configuration is not made in the deployment manifest, but in user-created `vm-import-controller-config` config map in the controller installation namespace; like described in https://github.com/kubevirt/vm-import-operator/blob/master/docs/design.md#common-templates.

Reply:
@jakub-dzon 

If the configuration is made in the ConfigMap, does that mean the user does not have to edit the `vm-import-operator` deployment manifest? Should I take those steps out?

If they are to be removed, how is the ConfigMap change applied? Does the user have to restart the pod or does it happen automatically?